tests/integration/models/reddit/test_modmail.py

```diff
@@ -0,0 +1,12 @@
+from praw.models import ModmailMessage
+
+from ... import IntegrationTest
+
+
+class TestModmailConversation(IntegrationTest):
+    def test_reply(self):
+        self.reddit.read_only = False
+        conversation = self.reddit.subreddit('all').modmail('ik72')
+        with self.recorder.use_cassette('TestModmailConversation.test_reply'):
+            reply = conversation.reply('A message')
+            assert isinstance(reply, ModmailMessage)
```

Comment:
While it is not 100% consistent, I try to make assertions outside of the use_cassette context manager that do not result in issue a request. Can you unindent one level here?